src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
To follow a more concise and conventional BEM approach, you might consider simplifying it to something like `navigation__item`
 
Block: Represents a standalone entity, such as a component.
Element: Represents a part of a block and is delimited by a double underscore (__).
Modifier: Represents a variation or state of a block or element and is delimited by a double hyphen (--).

More details about BEM can be found [here](https://mate-academy.github.io/style-guides/htmlcss.html#oocss-and-bem)